src/App.js

```diff
@@ -5,13 +5,14 @@ import store from './redux/store';
 import Books from './components/Books';
 import Categories from './components/Categories';
 import Navbar from './components/Navbar';
-import './App.css';
+import './Style/App.css';
+import styles from './Style/bookStoreApp.module.css';
 
 function App() {
   return (
     <Provider store={store}>
       <Router>
-        <div>
+        <div className={styles.pageContainer}>
           <Navbar />
           <Routes>
             <Route exact path="/" component={Books} />
```

Comment:
When starting the application the first time, it displays a blank page. It might be because you have used **component** instead of the **element**. Kindly fix this issue. ⬇️
![image](https://user-images.githubusercontent.com/33268927/224469371-b5c5ed6c-2153-4281-aeb0-a62bda5aef96.png)

Reply:
Thanks, I fix it: <Route exact path="/" element={<Navigate to="/books" />} />